src/components/button.component.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Button = (props) => (
+        <button className={`${props.wide}-button-${props.color}`}>{props.button}</button>
+)
+
+Button.propTypes = {
+        button: PropTypes.string.isRequired,
+        wide: PropTypes.bool,
```

Comment:
- [ ] Since no default props were implemented to the `wide` prop, we need to declare it as a required prop
```javascript
wide: PropTypes.bool.isRequired,
```